kitsune/settings.py

```diff
@@ -728,6 +730,10 @@ def JINJA_CONFIG():
     "SESSION_SERIALIZER", default="django.contrib.sessions.serializers.PickleSerializer"
 )
 
+
+# Our group with permissions to see hidden docs and message groups
+SUMO_STAFF = "Staff"
```

Comment:
Could we use `STAFF_GROUP` instead of `SUMO_STAFF`? I'll change https://github.com/mozilla/kitsune/pull/5947 to use whatever you use here.

Reply:
You got it.